connect/routes/fhir.py

```diff
@@ -107,11 +107,9 @@ async def post_fhir_data(
             do_retransmit=settings.nats_enable_retransmit,
             transmission_attributes={k: v for k, v in request.headers.items()},
         )
-        results = await workflow.run()
-
-        return workflow.set_response(response, results)
+        return await workflow.run()
     except Exception as ex:
-        raise HTTPException(status_code=500, detail=str(ex))
+        raise HTTPException(status_code=500, detail=ex)
```

Comment:
Reverted this to detail=ex because detail=str(ex) hadn't actually fixed the problem, which is now fixed in another spot in this PR.